hadoop-ozone/common/src/main/java/org/apache/hadoop/ozone/OzoneAcl.java

```diff
@@ -75,9 +77,20 @@ public OzoneAcl(ACLIdentityType type, String name, ACLType acl) {
    * @param name - Name of user
    * @param acls - Rights
    */
-  public OzoneAcl(ACLIdentityType type, String name, List<ACLType> acls) {
+  public OzoneAcl(ACLIdentityType type, String name, BitSet acls) {
+    Objects.requireNonNull(type);
+    Objects.requireNonNull(acls);
+
+    if(acls.cardinality() > ACLType.getNoOfAcls()) {
+      throw new IllegalArgumentException("Acl bitset passed has unexpected " +
+          "size. bitset size:" + acls.cardinality() + ", bitset:"
+          + acls.toString());
+    }
+
+    this.aclBitSet = new BitSet();
```

Comment:
this can be simplified as this.aclBitSet= acls.clone();

Reply:
done